packages/launchpad-ledger/src/demo/node.ts

```diff
@@ -1,6 +1,7 @@
 /* eslint-disable @typescript-eslint/naming-convention */
 import { toBase64 } from "@cosmjs/encoding";
 import { makeCosmoshubPath, makeSignDoc, StdFee, StdSignature } from "@cosmjs/launchpad";
+import LedgerTransport from "@ledgerhq/hw-transport-node-hid";
```

Comment:
What about using the orgininal transport name form https://github.com/LedgerHQ/ledgerjs/tree/master/packages/hw-transport-node-hid/src?